database-mysql/index.js

```diff
@@ -25,6 +25,10 @@ const Messages = connection.define('messages', {
   toId: Sequelize.INTEGER
 })
 
+const getUsers = () => {
+  return sequelize.query(`SELECT * FROM users`);
```

Comment:
this is okay but kinda the whole point of sequelize is to not have to write raw SQL query
better to use the Models methods like User.findAll() I believe it is